Tools/AP_Bootloader/bl_protocol.cpp

```diff
@@ -323,8 +323,10 @@ jump_to_app()
     rccDisableAPB1(~0);
 #endif
     rccDisableAPB2(~0);
-#if HAL_USE_SERIAL_USB == TRUE    
+#if HAL_USE_SERIAL_USB == TRUE
+#if !defined(STM32H730xx)
```

Comment:
My guess is that ChibiOS is missing the ifndef around STM32H730xx to disable rccResetOTG_FS. 
Might be work updating stm32h7xx/stm32_rcc.h for ChibiOS for this if that's the case.

Reply:
I fixed this in a slightly more general way